lib/ts/recipe/dashboard/api/listTenants.ts

```diff
@@ -16,22 +16,25 @@ import { APIInterface, APIOptions } from "../types";
 import Multitenancy from "../../multitenancy";
 import { ProviderConfig } from "../../thirdparty/types";
 
+type TenantListTenantType = {
+    tenantId: string;
+    emailPassword: {
+        enabled: boolean;
+    };
+    passwordless: {
+        enabled: boolean;
+    };
+    thirdParty: {
+        enabled: boolean;
+        providers: ProviderConfig[];
+    };
+    // This optional any is to allow deleting the key before sending the response to the frontend
+    coreConfig?: any;
```

Comment:
this should not be in the type output at all.. not sure why this is done this way.

Reply:
This was done because I was spreading the original tenant object and deleting core config. Deleting does not work for non optional types. Ive refactored to manually setting the properties instead of spreading so this is no longer an issue